cinnabar/plotting.py

```diff
@@ -55,13 +55,13 @@ def _master_plot(
         name of method associated with results, e.g. 'perses'
     target_name : string, optional
         name of system for results, e.g. 'Thrombin'
-    quantity : str, default = '$\Delta \Delta$ G'
+    quantity : str, default = '$\\Delta \\Delta$ G'
```

Comment:
I dislike this, but I dislike our readthedocs builds breaking even more. this code doesn't even get rendered in our docs.